docs/dependency-updates.md

```diff
@@ -183,6 +183,10 @@ Below is a list of dependencies that are locked down due to known issues with se
 
 - Peer-dependency tar-fs has high security vulnerability but this shouldn't affect us as far as we are aware of.
 
+- On October 27th, 2025, attempted to update puppeteer to 24.23.1 and @sparticuz/chromium to 141.0.0. However, this caused PDF visual regression tests to fail with pixel differences of 0.5-1.3% due to rendering engine changes in the newer Chromium version. The updates were reverted to maintain test stability.
+
```

Comment:
Just a note for future dependency-updaters: we expect some level of visual drift when updating chromium. Manually verify that any larger differences do not change the layout of docs (e.g. pushing text to the next page, breaking bulleted lists); if the changes are minimal, simply regenerate the reference images and the tests will pass.